app.config.js

```diff
@@ -201,9 +198,7 @@ module.exports = function (config) {
           '@sentry/react-native/expo',
           {
             organization: 'blueskyweb',
-            project: 'react-native',
-            release: VERSION,
-            dist: SENTRY_DIST,
+            project: 'app',
```

Comment:
Additional config is no longer supported here, you must use `SENTRY_RELEASE` and `SENTRY_DIST` env vars.